docs/platform/howto/byoc/add-customer-info-custom-cloud.md

```diff
@@ -36,7 +38,7 @@ a mandatory role that is always required as a primary support contact.
 
 3.  From the top navigation bar, select **Admin**.
 
-4.  From the left sidebar, select **Bring your own cloud**.
+4.  From the left sidebar, select <ConsoleLabel name="bringyourowncloud"/>.
```

Comment:
Can't comment directly on it, but line 66 also has this instruction in step 4, but without the icon. :)